api-catalog-package/src/main/resources/bin/start.sh

```diff
@@ -233,11 +254,13 @@ _BPX_JOBNAME=${ZWE_zowe_job_prefix}${CATALOG_CODE} java \
     -Dapiml.catalog.customStyle.headerColor=${ZWE_configs_apiml_catalog_customStyle_headerColor:-} \
     -Dapiml.catalog.customStyle.textColor=${ZWE_configs_apiml_catalog_customStyle_textColor:-} \
     -Dapiml.catalog.customStyle.docLink=${ZWE_configs_apiml_catalog_customStyle_docLink:-} \
-    -Dapiml.httpclient.ssl.enabled-protocols=${ZWE_components_gateway_apiml_httpclient_ssl_enabled_protocols:-"TLSv1.2"} \
+    -Dapiml.httpclient.ssl.enabled-protocols=${client_tls} \
     -Dspring.profiles.include=$LOG_LEVEL \
-    -Dserver.address=0.0.0.0 \
+    -Dserver.address=${ZWE_configs_zowe_network_server_listenAddresses_0:-${ZWE_zowe_network_server_listenAddresses_0:-"0.0.0.0"}} \
     -Dserver.ssl.enabled=${ZWE_configs_server_ssl_enabled:-true}  \
-    -Dserver.ssl.protocol=${ZWE_configs_server_ssl_protocol:-"TLSv1.2"}  \
+    -Dserver.ssl.protocol=${server_tls} \
+    -Dserver.ssl.ciphers=${ZWE_configs_zowe_network_server_tls_ciphers:-${ZWE_zowe_network_server_tls_ciphers:-}} \
```

Comment:
I am sorry, I meant ciphers. This property is setting ciphers for Tomcat and we don't have direct access to that part of the code. I think we can define them on the top of the script so it is better visible.

Reply:
This line allows it to be possible to define such a standard cipher list within "defaults.yaml" here https://github.com/zowe/zowe-install-packaging/blob/v2.x/staging/files/defaults.yaml

An advantage of this is so that the default ciphers are visible in 1 place.
Otherwise you'd have defaults for APIML, app-server, and ZSS all in separate places that people don't know about (which is the current state)

Reply:
Hi Sean, so if I understood correctly, there will be a new property `zowe_network_server_tls_ciphers` in this default.yaml, that's what you're proposing?

Reply:
@1000TurquoisePogs , I would like to mention that there is a PR that is clashing with your one - https://github.com/zowe/api-layer/pull/3689.

I understand you want to use the global configuration (I welcome it), but I'm not sure about a few things.

Why is the change not using the min tls? I assume we should construct 2 different values: protocol and enabledProtocols (see protocol=TLSv1.3, enabledProtocols=TLSv1.2,TLSv1.3). Then you are reusing the same structure as in the global configuration for the GW (and other components), even if we have a different structure of SSL configuration. Do you want to ensure that all components will use the same structure?

In all cases, we should merge your and min PR in the one new because they are modifying the same properties.

Reply:
After a discussion with @taban03 we have decided to make changes in PR #3689. @1000TurquoisePogs , could you please check if it could be ok from zour point of view? It is just an example for GW now.